concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,55 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3208	CENTER OR MIDDLE	Spatial relations	What's in the mid of a plane or sphere or equally far away from two ends.	Person/Thing	
+3209	MAKE A SPEECH	Speech and language	To give a speech in front of an audience	Action/Process	
+3210	KNIFE (FOR EATING)	Food and drink	A knife used while eating food.	Person/Thing	
+3211	HOPE (FAITH)	Emotions and values	The belief in the possibility that something one wants will happen.	Person/Thing	
+3212	HOPE (SOMETHING)	Emotions and values	To want something to happen.	Action/Process	
+3213	HIRE OR RENT	Social and political relations	To employ a person or to pay for using an object.	Action/Process	
+3214	CRAWL OR CREEP	Motion	To move using ones hands and knees or in a prone position with the body resting on or close to the ground.	Action/Process	
+3215	CREEP	Motion	To move in a prone position with the body resting on or close to the ground.	Action/Process	
+3216	DRIP OR DROP SOMETHING	Motion	To fall in globules or small portions, as water or other liquid or to let something fall down.	Action/Process	
+3217	TIN OR TINPLATE	Basic actions and technology	A metallic element with symbol Sn or a thin sheet of steel coated with tin to prevent rusting.	Person/Thing	
+3218	SPRAY	Motion	To cause a liquid to fly through the air in small droplets.	Action/Process	
+3219	SPLASH OR SPRAY	Motion	To cause a liquid to spatter about [WN:365] or to cause it to fly through the air in small droplets.	Action/Process	
+3220	MATERNAL UNCLE	Kinship	A brother of someone's mother.	Person/Thing	
+3221	PLANTAGO	Agriculture and vegetation	A plant of the genus Plantago, with a rosette of sessile leaves about 10 cm long with a narrow part instead of a petiole, and with a spike inflorescence with the flower spacing varying widely among the species.	Person/Thing	
+3222	TWIG	Agriculture and vegetation	A branch that doesn't arise from the trunk but from another branch.	Person/Thing	
+3223	BRANCH OR TWIG	Agriculture and vegetation	A woody part of a tree arising from the trunk or from another branch.	Person/Thing	
+3224	WRING	Basic actions and technology	To take a cloth and twist its ends into different directions in order to dry it.	Action/Process	
+3225	SQUEEZE OR WRING	Basic actions and technology	To apply pressure from more than one side or to twist a cloth's ends into different directions.	Action/Process	
+3226	FASTEN OR SQUEEZE	Basic actions and technology	To close tightly or to apply pressure from more than one side.	Action/Process	
+3227	SPHERE	Spatial relations	Any spherical object.	Person/Thing	
+3228	BALL OR SPHERE	Spatial relations	An object, generally spherical, used for playing games, or any spherical object.	Person/Thing	
+3229	STINK	Sense perception	To emit an unpleasant odour.	Action/Process	
+3230	DECEIVE	Cognition	To cause someone to believe an untruth; to practice trickery or fraud.	Action/Process	
+3231	DEITY	Religion and belief	A supernatural, typically immortal being with superior powers.	Person/Thing	
```

Comment:
I see, this is related even to Leibniz in terms of the concept GOD above. To make sure people to confuse, I'd have something as GOD (ABRAHAMIC) in the gloss above.

Reply:
But we should check how things are mapped right now -- are we sure words now mapped to GOD don't actually relate to the DEITY concept?

Reply:
I moved many of the entries to a new concept DEITY. I only left those at GOD that were published before 1900 or have "God" instead of "god".

Reply:
Great :+1: